### PR TITLE
Loading JS once in the given cell

### PR DESCRIPTION
**Pull request by jlstevens**
PR using the information now exposed in https://github.com/holoviz/pyviz_comms/pull/87 to avoid repeatedly loading JS in the same cell (e.g. if multiple extensions are in the first cell of a notebook).

### CODE REVIEW

**Comment by philippjfr** [comment]
Does not seem sufficient, you'll also have to handle these lines:

        for r in [r for r in resources if r != 'holoviews']:
            Store.renderers[r].load_nb(inline=p.inline)
        Renderer.load_nb(inline=p.inline)

and we probably also need to think about adding warning checking that if `same_cell_execution` is true and `'plotly' not in panel.extension._loaded_extensions` that the user has to make sure they run `hv.extension('plotly')` or `pn.extension('plotly')` first.

File: holoviews/ipython/__init__.py [hunk 1]
```diff
@@ -183,20 +183,23 @@ def __call__(self, *args, **params):
         if hasattr(config, 'comms') and comms:
             config.comms = comms
 
-        for r in [r for r in resources if r != 'holoviews']:
-            Store.renderers[r].load_nb(inline=p.inline)
-        Renderer.load_nb(inline=p.inline)
+        if not same_cell_execution:
```
Comment (philippjfr) [inline holoviews/ipython/__init__.py:186]: Variable is not defined yet, you define it below.

Comment (jlstevens) [reply holoviews/ipython/__init__.py:186]: Oops! Well spotted.

Comment (jlstevens) [reply holoviews/ipython/__init__.py:187]: Fixed in [b946bbf](https://github.com/holoviz/holoviews/pull/5249/commits/b946bbfa895fc49678725bad5f95e0d20a918c6d)

**Comment by maximlt** [comment]
I've merged master and the tests all passed 🎉 Ready to be merged then I think @jlstevens

**Comment by jlstevens** [comment]
Thanks for fixing the tests! @philippjfr please merge if you think this PR is ready.

**Pull request merged**